src/Germany/Mews.Fiscalizations.Germany/V2/FiskalyClient.cs

```diff
@@ -73,10 +72,12 @@ public Task<ResponseResult<Tss>> GetTssAsync(AccessToken token, Guid tssId)
             );
         }
 
-        public Task<ResponseResult<IEnumerable<Tss>>> GetAllTSSsAsync(AccessToken token)
+        public Task<ResponseResult<IEnumerable<Tss>>> GetAllTSSsAsync(AccessToken token, TssStates states)
         {
+            var flags = Enum.GetValues(typeof(TssStates)).Cast<TssStates>().Where(s => states.HasFlag(s)).ToNonEmptyOption();
+            var requestParams = flags.Map(f => String.Join("&", f.Select(s => $"states={s.ToString().ToUpper()}")));
```

Comment:
This would break if we send in `None`. Maybe we could do a nonEmpty enumerable of enum, rather than a flag enum.